cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -167,3 +167,15 @@ func RemovePrivacyLogTag(message string) string {
 
 	return message
 }
+
+// PostProcessDErrorForNetworkFlag processes DError for network flag
+func PostProcessDErrorForNetworkFlag(action string, err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
+		if derr.CausedByErrType("networkResourceDoesNotExist") && network == "" {
+			w.LogWorkflowInfo("A VPC network is required for running %v," +
```

Comment:
Does this even build? where the closing bracket for the func call?

Reply:
It has no problem, I don't get...

Reply:
Something was off with github showing this change, didn't see lines below.